documentation/Troubleshooting.md

```diff
@@ -83,285 +92,391 @@ Traceback (most recent call last):
+- Validate the cluster's node configuration before deployment to ensure the number of workers is correctly set.
+- Regularly check the system's configuration and update it as necessary.
+
+>**Note**  
+>If you resolve the problem, consider [opening a new PR](https://github.com/Netcracker/KubeMarine/pulls) to document your solution, Our team will appreciate it!
 
 
 ## KME0002: Remote group exception
 
-Shell error:
+### Description
+This error occurs when a bash command executed on a remote cluster host terminates unexpectedly with a non-zero exit code. In this case, the command `'apt install bad-package-name'` failed with exit code 127, indicating that the `apt` command was not found on the remote node.
 
-```
+### Alerts
+Not applicable.
+
+### Stack trace(s)
+```text
 FAILURE!
 TASK FAILED xxx
 KME0002: Remote group exception
 10.101.10.1:
-	Encountered a bad command exit code!
-	
-	Command: 'apt install bad-package-name'
-	
-	Exit code: 127
-	
-	=== stderr ===
-	bash: apt: command not found
-```
-
-An error indicating an unexpected runtime bash command exit on a remote cluster host. This error 
-occurs when a command is terminated unexpectedly with a non-zero error code.
-
-The error prints the status of the command execution for each node in the group on which the bash command 
-was executed. The status can be a correct result (shell results), a result with an error 
-(shell error), as well as a [timeout](#command-did-not-complete-within-a-number-of-seconds) error.
-
-To fix it, first try checking the nodes and the cluster with 
-[IAAS checker](Kubecheck.md#iaas-procedure) and [PAAS checker](Kubecheck.md#paas-procedure). If you 
-see failed tests, try fixing the cause of the failure. Make sure that you do everything according to 
-the instructions in the correct sequence and correctly fill the inventory and other dependent
-files. If the error persists, try to figure out what might be causing the command to fail on remote 
-nodes and fix by yourself as much as possible.
-
-If you still can't resolve this error yourself, start 
-[a new issue](https://github.com/Netcracker/KubeMarine/issues/new) and attach a description of the 
-error with its stacktrace. We will try to help as soon as possible.
-
-
-### Command did not complete within a number of seconds
-
-```
+  Encountered a bad command exit code!
+  
+  Command: 'apt install bad-package-name'
+  
+  Exit code: 127
+  
+  === stderr ===
+  bash: apt: command not found
+```
+
+### How to solve
+1. Run the [IAAS checker](Kubecheck.md#iaas-procedure) and [PAAS checker](Kubecheck.md#paas-procedure) to ensure that the infrastructure and platform are functioning properly.
+2. Inspect the node where the error occurred and verify the presence of the required package manager. 
+3. If the package manager is not available, ensure that the correct package management tool is installed on the node or adjust the command to use the appropriate tool (e.g., `yum` for RHEL-based systems).
```

Comment:
These steps (and recommendations below) are specific to one of many such KME0002 problems as I understand. I am not sure we need to add steps related to package management specifically, but in general for failed commands.  However, we probably can add something example-like:
```markdown
2. Inspect the node where the error occurred. In our particular example, it is required to check the presence of the required package manager and install it if it is missing
```

Reply:
Okay added the general line